pkg/k8sutil/pvc.go

```diff
@@ -104,7 +106,7 @@ func GenerateCSIPVC(storageclass string, pvc *corev1.PersistentVolumeClaim) *cor
 	return csiPVC
 }
 
-func CreatePVC(c *k8s.Clientset, pvc *corev1.PersistentVolumeClaim, t int) (*corev1.PersistentVolume, error) {
+func WaitForRBDImage(c *k8s.Clientset, pvc *corev1.PersistentVolumeClaim, t int) (*corev1.PersistentVolume, error) {
```

Comment:
This method still creates the PVC, I think this was correct
```suggestion
func CreatePVC(c *k8s.Clientset, pvc *corev1.PersistentVolumeClaim, t int) (*corev1.PersistentVolume, error) {
```

Reply:
Thanks @travisn , Updated now.